src/api/index.ts

```diff
@@ -59,4 +70,19 @@ export function odataType(rawOdataType: string, friendlyType?: string, typePrope
       }
     };
   };
-}
\ No newline at end of file
+}
+
+/**
+ * Try converting a raw object or instance into an instance of the specified model class.
+ *
+ * @param modelClass The model class to convert into.
+ * @param rawOrInstance
+ * + `null`, `undefined` or instances of `modelClass` are returned as is.
+ * + Anything else is converted into the model class using `modelClass.create`.
+ */
+export function tryCreateModel<M, T extends ModelInstance<M> | null | undefined>(modelClass: M, rawOrInstance: T): T {
```

Comment:
A `tryACTION` model usually indicates that the method returns `false` if the action `ACTION` is not successful. Probably just rename the function to `createModel`.